plugins/org.jkiss.dbeaver.ext.dameng.ui/OSGI-INF/l10n/bundle.properties

```diff
@@ -0,0 +1,2 @@
+editor.source.ddl.name=DDL
+editor.source.ddl.description=DDL
```

Comment:
And also you need to add OSGI to the plugins/org.jkiss.dbeaver.ext.dameng.ui/build.properties